source/features/github-bugs.css

```diff
@@ -0,0 +1,25 @@
+/* Place Releases/Tags selector on the same line as `Draft new release` on releases page #2325 */
+.subnav .d-flex.flex-md-row.flex-justify-between.flex-md-items-center {
+	float: left;
+}
+
+/* Align fork popup's header #3352 */
+[aria-label^='Fork'] .Box-title {
+	position: static; /* The title wrongly overlaps the Close button, making it impossible to click */
+	min-height: 0;
+	margin: -5px 0;
+}
+
+/* Reposition committer's avatar on PR commits #3444 */
+.pull-request-tab-content .commit .AvatarStack {
+	display: inline-block;
+	margin-bottom: -4px;
+	margin-right: -4px;
+}
+
+/* Add missing border radius to collapsed PR files #3444 */
+.js-blob-wrapper,
+.file:not(.open) .file-header.file-header--expandable {
```

Comment:
I added `background: red` to see what elements this affects. It affects a lot of unrelated elements

![sdfaaaaaasad](https://user-images.githubusercontent.com/1402241/89803328-94d02600-db2a-11ea-8a5d-c2011c851af9.gif)

Reply:
> a lot of unrelated elements

![](https://user-images.githubusercontent.com/44045911/89803905-09748600-db66-11ea-845c-1b615a6570ac.png)

Reply:
> It appears to _affects a lot of unrelated elements_ because it applies to the wrapper

:d'oh: